dotnetAvalonia/.vscode/tasks.json

```diff
@@ -226,7 +230,7 @@
                 "-P",
                 "${config:torizon_debug_ssh_port}",
                 "-pr",
-                "${workspaceFolder}/bin/Debug/net6.0/linux-x64/publish",
+                "${workspaceFolder}/bin/Debug/net8.0/linux-x64/publish",
```

Comment:
Here `${workspaceFolder}/bin/Debug/net8.0/linux-amd64/publish` here, as we use `amd64` to refer to `x86` on the templates

Reply:
I can't change this because is the way that `dotnet publish` works, from the .NET ecosystem the x86-64 or amd64 is x64